### ISSUE TITLE
Oddity in HT3D with thin obstruction

### ISSUE THREAD

**Issue by johodges** [firemodels/fds#12262]
I adapted the checkerboard case to align with one of Rachel Wasson's cases from her thesis. I noticed an odd behavior in the temperatures being predicted with HT3D. The emissivity on the underside of the ceiling varies with the checkerboard pattern and the emissivity on the top side is constant. On the underside of the ceiling we see a smooth surface temperature profile; however, we see a checkerboard pattern to the temperature on the top of the ceiling.

[wasson50_1p5_cb.txt](https://github.com/firemodels/fds/files/13536617/wasson50_1p5_cb.txt)

![image](https://github.com/firemodels/fds/assets/17127980/1cc3e503-3b19-4ad6-bd61-ed5491ecac61)

**Comment by mcgratta** [comment]
Glenn, I compiled smokeview on burn to look at this case on Nomachine. When I load a boundary file, I get a core dump every time. All other quantities are OK.

**Comment by gforney** [comment]
I'll take a look

**Comment by gforney** [comment]
could you type smokeview -v and tell me what version you are using

**Comment by mcgratta** [comment]
```
[mcgratta@burn Test]$ smokeview -v

Smokeview   - Dec  2 2023

Revision         : SMV-6.8.0-1453-gb7ab6fa66-master
Revision Date    : Fri Dec 1 14:22:54 2023 -0500
Compilation Date : Dec  2 2023 13:35:57
Compiler         : Intel(R) oneAPI DPC++/C++ Compiler 2022.2.1 (2022.2.1.20221020)
Checksum(SHA1)   : 68220c7f1d1eb7d0ac46d51d6cd96ea91d103acd
Platform         : LINUX64
Smokeview path   : /home4/mcgratta/firemodels/smv/Build/smokeview/intel_linux_64/smokeview_linux_64
Bin directory    : /home4/mcgratta/Test/
```

**Comment by gforney** [comment]
update your repo and try again.

**Comment by johodges** [comment]
Thanks for looking into it. The part that is still confusing to me is how we are getting a checkerboard pattern on the backside when we have a fairly uniform thermal profile on the bottom and a uniform surface boundary on the back side (specified h and emissivity as constant). It's unclear to me how convection in this case would lead to this non-uniformity.  I'll take a look another look at the results to see if I can resolve what I am seeing in my head.

**Comment by mcgratta** [comment]
Is this still odd?

**Comment by johodges** [comment]
Thanks for bumping this. Looking into it more today.

**Comment by johodges** [comment]
I think I figured out the source of the oddity I was seeing in the HT3D results. The issue is that the grid used in the 1-D heat transfer solution in the lateral directions does not consider the variability in surface definition on that dimension (i.e., the checkerboard pattern in this case). As a result, the number of nodes in the lateral conduction is insufficient to resolve the spatial variability. 

The simplified case here demonstrates the difference. The case is 0.6m tall, 1.2m wide, 0.001 m thick. Checkerboard pattern of 0.02 m and gas phase grid cell of 0.01 m in-plane, and 0.001 m in the thickness. The default profile (left) through the longer dimensions is too coarse to resolve the variation we are seeing in the checkerboard pattern; whereas, the CELL_SIZE=0.001 profile (right) resolves the variation reasonably well. We ran this same case in Abaqus to check the agreement of the resolved gas. Overall the results were fairly similar with the range of surface temperatures observed being a bit larger in Abaqus than in FDS.

We can recommend the users specify CELL_SIZE if invoking HT3D, but I am wondering if there is something we can do on the backend to improve the default noding.

[radiant_panel_solid_fixed_cell_size.txt](https://github.com/firemodels/fds/files/13690235/radiant_panel_solid_fixed_cell_size.txt)

![checkerboard_conduction](https://github.com/firemodels/fds/assets/17127980/ba186779-b13d-470d-b769-70042515f57d)
